os/board/common/README.md

```diff
@@ -17,7 +17,7 @@ Note that the values for FLASH_PART_SIZE, FLASH_PART_TYPE and FLASH_PART_NAME mu
 
 FLASH_PART_SIZE "16,48,192,32,512,2400,1536,1536,1400,8,512,"  
 FLASH_PART_TYPE "none,none,none,none,none,none,none,none,smartfs,config,none,"  
-FLASH_PART_NAME "bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,user,nvram,sssrw,"  
+FLASH_PART_NAME "bl1,sssro,bl2,sssfw,wlanfw,os,factory,ota,userfs,nvram,sssrw,"
 
```

Comment:
Other lines in this file also use the existing nomenclature i.e. "user" instead of "userfs".
Could you take care of those as well?
For Eg - Lines 31 `user    | smartfs | 1400`
70` Below steps creates ROMFS partition with size 400KB at next of user partition.`
72 `    1. Split user partition size from (1400) to (1000, 400) in FLASH_PART_SIZE` 
84 `'    3. Append "rom" at next of *user* to FLASH_PART_NAME`

Reply:
Modify